installing/installing_with_agent_based_installer/prepare-pxe-infra-agent.adoc

```diff
@@ -0,0 +1,38 @@
+:_content-type: ASSEMBLY
+[id="prepare-pxe-infra-agent"]
+= Preparing PXE booting infrastructure for {product-title}
+include::_attributes/common-attributes.adoc[]
+:context: prepare-pxe-infra-agent
+
+toc::[]
+
+Use the following procedures to create the assets needed to PXE boot an {product-title} cluster using the Agent-based Installer.
+
+The procedures create assets to deploy a single-node {product-title} installation. You can use these procedures as a basis and modify according to your requirements.
+
+[id="prerequisites_prepare-pxe-infra-agent"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+// Downloading the Agent-based Installer
+include::modules/installing-ocp-agent-download.adoc[leveloffset=+1]
+
+// Creating the preferred configuration inputs
```

Comment:
This by default says `Use this procedure to create the preferred configuration inputs used to create the agent image.` But I think it should be updated to `Use this procedure to create the preferred configuration inputs used to create the agent image or PXE files.`

Reply:
I'll add conditional statements so that it only mentions the agent image when this module shows up in the regular install doc, and only mentions PXE assets when this module is included in this PXE doc.